### ISSUE TITLE
Compat with BaseMetals [1.12.2] Also crashes the game.

### ISSUE THREAD

**Issue by KarinaFrost** [P3pp3rF1y/AncientWarfare2#973]
I finally got Github to load. Anyways, I suppose I'll include our discord DMs for help. 
[crash-2019-06-19_09.58.29-server.txt](https://github.com/P3pp3rF1y/AncientWarfare2/files/3308685/crash-2019-06-19_09.58.29-server.txt)
[crash-2019-06-19_09.49.27-server.txt](https://github.com/P3pp3rF1y/AncientWarfare2/files/3308686/crash-2019-06-19_09.49.27-server.txt)
[crash-2019-06-19_22.55.23-server.txt](https://github.com/P3pp3rF1y/AncientWarfare2/files/3308687/crash-2019-06-19_22.55.23-server.txt)
[crash-2019-06-19_10.47.08-server.txt](https://github.com/P3pp3rF1y/AncientWarfare2/files/3308688/crash-2019-06-19_10.47.08-server.txt)
![image](https://user-images.githubusercontent.com/6021874/59817449-86e79200-92e5-11e9-8d8c-0caeb09c468e.png)

So basically, I gave a worker a hoe from BaseMetals (And recently ModernMetals) and the game crashed. Related Block was Crop Farm, and Fish Farm. PROBABLY safe to assume it happens to other farm blocks too. Strangely, it doesn't happen with the Quarry. NORMALLY, I'd just play with code and bandaid it to the point of working, but I have no skill with Java aside from 'Hello World!' So I come to you. Will do more testing on request.

**Comment by ThatPolishKid99** [comment]
Hey there! It’s been a while, but I finally got around to fixing this for the MMD guys. In theory, it should be fixed in their next release, barring extraneous circumstances.

**Comment by ThatPolishKid99** [comment]
Fixed in latest release...When MMD releases their stuff, honestly not sure if they've done that or not
